include/snowhouse/exceptions.h

```diff
@@ -97,6 +99,14 @@ namespace snowhouse
   catch (...) \
   { \
     SNOWHOUSE_TEMPVAR(wrong_exception) = true; \
+    if (auto eptr = std::current_exception()) { \
+      try { \
+      std::rethrow_exception(eptr); \
+      } catch (std::exception& e) { \
+      SNOWHOUSE_TEMPVAR(more_info) = true; \
+      SNOWHOUSE_TEMPVAR(info_string) = e.what(); \
+      } catch (...) {} \
```

Comment:
Indentation has gone wrong here. (This part is not checked by clang-format.)

Reply:
Oh right, somehow I stopped paying attention to that since clang-format always deals with that for me otherwise

Reply:
Unfortunately I didn't get clang-format to handle multiline macro definitions as I like them, so it's turned off for those sections.

Reply:
Ah okay :)